packages/grafana-data/src/transformations/transformers/histogram.ts

```diff
@@ -301,7 +312,15 @@ function histogram(
 /**
  * @internal
  */
-export function histogramFieldsToFrame(info: HistogramFields): DataFrame {
+export function histogramFieldsToFrame(info: HistogramFields, theme?: GrafanaTheme2): DataFrame {
+  if (!info.bucketMin.display) {
+    const display = getDisplayProcessor({
+      field: info.bucketMin,
+      theme: theme ?? createTheme(),
```

Comment:
theme does not really matter since we do not use the displayProcessor for colors (in this case)